modAionImpl/src/org/aion/zero/impl/AionBlockchainImpl.java

```diff
@@ -58,13 +58,13 @@
 import org.aion.rlp.RLP;
 import org.aion.types.AionAddress;
 import org.aion.util.bytes.ByteUtil;
+import org.aion.util.bytes.Hash256;
```

Comment:
For Hash256,
I would prefer it to stay in the org.aion.util.types package.

Reply:
Done.